src/main/java/io/nats/client/impl/NatsConnectionReader.java

```diff
@@ -53,7 +53,7 @@
 
     private Mode mode;
 
-    private NatsMessage incoming;

```

Comment:
converted to packaged scope NatsMessage.IncomingBuilder. Considered calling it a factory, still am ;)